src/main/scala/net/mikolak/travesty/VizGraphProcessor.scala

```diff
@@ -19,7 +19,8 @@ class VizGraphProcessor(typeNameSimplifier: TypeNameSimplifier) {
     def implOf(v: Vertex) = v.asScala.property(properties.node.StageImplementation).value()
 
     val implNames = in.V().l().foldRight(Map.empty[AkkaStage, String]) { (v, map) =>
-      val baseName = v.asScala.property(properties.node.StageName).value()
+      // @fixme default stage name might not be the best solution
+      val baseName = v.asScala.property(properties.node.StageName).orElse("default stage name")
```

Comment:
"default stage name" is totally fine :). However, this should be loaded from the config, optimally in a new subsection (`render` ?), since it doesn't really seem like any of the existing ones fit.

Here's the config CC:
https://github.com/mikolak-net/travesty/blob/master/src/main/scala/net/mikolak/travesty/setup/TravestyConfig.scala
and here's the wiring class where the config gets injected:
https://github.com/mikolak-net/travesty/blob/master/src/main/scala/net/mikolak/travesty/setup/Wiring.scala

You can also read about the general concept [here](https://mikołak.net/blog/2016/type-safe-config-ficus-macwire.html), but honestly, you don't really need to - it's simple enough to follow by example.